web/src/components/AirgapUploadProgress.jsx

```diff
@@ -338,4 +338,4 @@ AirgapUploadProgress.defaultProps = {
   progress: 0,
 };
 
-export default withRouter(AirgapUploadProgress);
+export default AirgapUploadProgress;
```

Comment:
most of the changes are removals of `withRouter` calls. This decorator adds several react-router related props to the component. Several components use this decorated but not the props.